src/wallet/rpcdump.cpp

```diff
@@ -561,7 +561,11 @@ UniValue dumpwallet(const JSONRPCRequest& request)
             "dumpwallet \"filename\"\n"
             "\nDumps all wallet keys in a human-readable format.\n"
             "\nArguments:\n"
-            "1. \"filename\"    (string, required) The filename\n"
+            "1. \"filename\"    (string, required) The filename (if not full path, relative to bitcoind path)\n"
```

Comment:
The text is a bit clunky here. Perhaps "The filename with path (either absolute or relative to bitcoind)" is better?